src/main/java/com/doongji/nestalk/service/user/UserService.java

```diff
@@ -59,4 +61,11 @@ public String findEmailByNameAndPhone(String name, String phone) {
                 .map(User::getEmail)
                 .orElseThrow(() -> new NotFoundException(User.class, name, phone));
     }
+
+    @Transactional
+    public User userUpdate(Long userId,UserUpdateRequest updateRequest) {
+        User findUser = userRepository.findById(userId).orElseThrow(()->new NotFoundException("Not Found User"));
```

Comment:
```suggestion
        User findUser = userRepository.findById(userId).orElseThrow(() -> new NotFoundException("Not Found User"));
```
예외처리 메시지를 더 디테일하게 써주면 좋을 것 같아요.